tensorboard/plugins/hparams/hparams_plugin.py

```diff
@@ -168,11 +168,9 @@ def list_metric_evals_route(self, request):
                 raise werkzeug.exceptions.NotFound("Scalars plugin not loaded")
             return http_util.Respond(
                 request,
-                json.dumps(
-                    list_metric_evals.Handler(
-                        ctx, request_proto, scalars_plugin, experiment_id
-                    ).run()
-                ),
+                list_metric_evals.Handler(
```

Comment:
Fixes the server response so that metric charts can be rendered for session groups with NaNs